.github/workflows/ci.yml

```diff
@@ -51,7 +51,7 @@ concurrency:
 
 jobs:
   maven-checks:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Just for forward-stability. Or some other reason?

Reply:
https://github.com/microsoft/mssql-docker/issues/899